cdk/cdk-resources/project/.projenrc.js

```diff
@@ -0,0 +1,49 @@
+const { awscdk } = require('projen');
+const { ReleaseTrigger } = require('projen/lib/release');
+const project = new awscdk.AwsCdkConstructLibrary({
+  author: 'MongoDB',
+  authorAddress: 'https://mongodb.com',
+  authorName: 'MongoDB',
+  cdkVersion: '2.1.0',
+  defaultReleaseBranch: 'main',
+  majorVersion: 1,
+  releaseToNpm: true,
+  releaseTrigger: ReleaseTrigger.manual(),
+  docgen: true,
+  name: 'mongodb-atlas-project',
+  repositoryUrl: 'https://github.com/mongodb/mongodbatlas-cloudformation-resources.git',
+  sampleCode: false,
+  publishToPypi: {
```

Comment:
@sangramPeerI L16-L28 needs to be removed. We are only publishing to npm.

Reply:
@SuperMohit  Ok. will remove it.